package.json

```diff
@@ -39,6 +40,7 @@
     "jasmine-core": "2.7.0",
     "karma": "1.7.0",
     "karma-babel-preprocessor": "6.0.1",
+    "karma-browserify": "^5.1.1",
```

Comment:
the test runner now creates a browserify bundle for every component as it's running the test.